src/utils/validation.ts

```diff
@@ -360,6 +360,8 @@ export const beaconMetadataSchema = z.record(
     pricingCoverage: z.record(z.string()), //TODO must be present in pricingCoverage
     decimalPlaces: z.number().optional(),
     logos: z.array(z.string()).optional(),
+    prefix: z.string().optional(),
+    postfix: z.string().optional(),
```

Comment:
Does pre/postifx have any other additional feature worth validating? For instance, length === 1, etc...

Reply:
Hey, no, although we'll probably have to add general validation checking for dapis->dapiMetadata and beacons->beaconMetadata.

Reply:
I can probably do that quick.

Reply:
I've added reference checking for dapis->dapiMetdata, and on first run we were missing quite a few, so it works nicely.
It prints out a template for missing data, which I think will be very helpful.